less/mixins.less

```diff
@@ -1 +1,19 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+
+.button {
+    background: @eastern-blue;
+    width: 100%;
+    height: 3.5rem;
+    display: flex;
+    justify-content: center;
+    .border-radius();
+    color: @white;
+    font-size: 2rem;
+    align-items: center;
+}
+
+.border-radius (@radius: 5px) {
+    -webkit-border-radius: @radius;
+    -moz-border-radius: @radius;
+    border-radius: @radius;
```

Comment:
I like this that you gave the parameter a default value here! and are accounting for all browsers with `webkit` solutions